catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -12,24 +12,89 @@ spec:
   description: >-
```

Comment:
Yep. I had the same conversation some weeks ago because I wanted also to do small changes on something not in  use yet but then the conclusion was that we should always create a new version.

Reply:
Just rephrasing what I said. The conclusion was in the conversation I had (maybe with Scott) - Indeed, this was not a team consensus.

Reply:
Hmm, good point. But I vaguely remember a similar conversation where the conclusion was that if it wasn't used yet, it's not needed. And it would make sense - imho the reason for new versions is to avoid breaking some existing workflow. But maybe my memory is failing me :) In my opinion adding a new version here would just add more clutter with no benefit. I guess what matters is if it's ok with you as reviewers :)

Reply:
I think in cases like this one, this would only help to create/enforce the habbit as nothing is yet using it.  Anyway for the time being, all good changing the same ver.